src/utils/analytics/index.js

```diff
@@ -107,86 +109,20 @@ export const parseSearchUrl = (queryUrl) => {
     return parsedUrl;
 };
 
-export const EVENT_TYPES = Object.freeze({
-    ERROR: (action, type, status = 500) => ({
-        category: "error",
-        action: `error-${action}`,
-        label: `ERROR: ${type}`,
-        value: status,
-    }),
-    SUCCESS: (action, label) => ({
-        category: "success",
-        action,
-        label,
-    }),
-    OTHER: {
-        category: "other",
-        action: "another-event-type",
-    },
-});
-
-export const TIMED_ACTIONS = Object.freeze({
-    OFFER_CREATE: {
-        category: "offer",
-        variable: "offer/create",
-    },
-    OFFER_SEARCH: {
-        category: "offer",
-        variable: "offer/search",
-    },
-    OFFER_HIDE: {
-        category: "offer",
-        variable: "offer/hide",
-    },
-    OFFER_DISABLE: {
-        category: "offer",
-        variable: "offer/disable",
-    },
-    OFFER_ENABLE: {
-        category: "offer",
-        variable: "offer/enable",
-    },
-    COMPLETE_REGISTRATION: {
-        category: "registration",
-        variable: "registration/complete",
-    },
-    APPLICATION_SEARCH: {
-        category: "application",
-        variable: "application/search",
-    },
-    APPLICATION_APPROVE: {
-        category: "application",
-        variable: "application/approve",
-    },
-    APPLICATION_REJECT: {
-        category: "application",
-        variable: "application/reject",
-    },
-    APPLICATION_SUBMIT: {
-        category: "application",
-        variable: "application/submit",
-    },
-    COMPANY_OFFERS_FETCH: {
-        category: "company_offers",
-        variable: "company_offers/fetch",
-    },
-    OTHER: {
-        category: "other",
-        variable: "other",
-    },
-    UNKNOWN: {
-        category: "unknown",
-        variable: "unknown",
-    },
-});
-
 /**
- * Object mapping search filters to GA dimensions' indexes
+ * Records a visit to an offer's page
+ *
+ * @param {*} companyName Name of the company who owns the offer
+ * @param {*} offerTitle Title of the offer
+ * @param {*} offerId ID of the offer
  */
-export const DIMENSION_IDS = Object.freeze({
-    jobType: "dimension1",
-    jobMinDuration: "dimension2",
-    jobMaxDuration: "dimension3",
-    fields: "dimension4",
-    technologies: "dimension5",
-});
+export const recordOfferVisit = (offerId, offerTitle, companyName) => {
+    if (!offerId) return; // Invalid offer, do not set dimensions
+
+    const dimensions = {};
+    dimensions[DIMENSION_IDS.companyName] = companyName;
+    dimensions[DIMENSION_IDS.offerTitle] = offerTitle;
+    ReactGa.set(dimensions);
```

Comment:
```suggestion
    ReactGa.set({
[DIMENSION_IDS.companyName]: companyName,
[DIMENSION_IDS.offerTitle]: offerTitle
});
```

Reply:
That's exactly what I wanted but didn't know how :p